### DISCUSSION TITLE
Bradford Bin names

### DISCUSSION THREAD

**Discussion by octotheoli** [mampfes/hacs_waste_collection_schedule#1928] [Q&A]
I have HACS waste collection installed. I have the sensor but i dont know the names for the bin type. Ill attach my config file for help.

```waste_collection_schedule:
  sources:
    - name: bradford_gov_uk
      args:
        uprn: 100051201116
 
sensor:
  - platform: waste_collection_schedule
    name: Home
    count: 2
    leadtime: 7
    value_template: 'in {{value.daysTo}} days'
    date_template: value.daysTo
    add_days_to: True
    types:
      - General
      - Recycling
    details_format:
        appointment_types```

**Comment by 5ila5** [comment]
bradford_gov_uk returns waste types `REFUSE`, `RECYCLING`, `GARDEN`, it's best to look in your HA calendar tab which events are being reported. bradford_gov_uk currently only accepts String UPRNs and not integer UPRNs so you need to add `"` around your UPRN (`uprn: "100051201116"`) I fixed this behavior in #1935, which will be part of the next release.
